articles/best-practices/custom-db-connections/anatomy.md

```diff
@@ -0,0 +1,65 @@
+:::
+
+You typically create and configure custom database connections using the [Auth0 Dashboard](/connections/database/custom-db/create-db-connection#step-1-create-and-configure-a-custom-database-connection). You create a database connection and then toggle **Use my own database** to enable editing of the database action scripts. A custom database connection can also be created and configured with the Auth0 [Management API](/api/management/v2#!/Connections/post_connections) and the `auth0` strategy. 
+
+![Enable Use Own Database Option](/media/articles/dashboard/connections/database/connections-db-settings-custom-1.png)
+
+As shown below, you use custom database connections as part of the Universal Login workflow to obtain user identity information from your own legacy identity store for authentication or user import, referred to as *legacy authentiation*.
+
+![Custom Database Connections Flow](/media/articles/connections/database/custom-database-connections.png)
+
+In addition to artifacts common for all database connections types, a custom database connection allows you to configure action scripts (custom code used when interfacing with legacy identity stores). The scripts you choose to configure depend on whether you are creating a connection for legacy authentication or for automatic migration. 
+
+::: panel Best Practice
+You can use action scripts as anonymous functions, however anonymous functions make it hard to debug when it comes to interpreting the call-stack generated as a result of any exceptional error conditions. For convenience, we recommend providing a function name for each action script and have supplied some [recommended names](/best-practices/custom-db-connections/exeution#recommended-script-names).
+:::
+
+In a legacy authentication scenario, no new user record is created; the user remains in the legacy identity store and Aut0 uses the identity it contains when authenticating the user.
+
+::: note
+Custom database connections are also used outside of the Universal Login workflow. For example, a connection's [`changePassword` action script](#change-password) is called when a password change operation occurs for a user that resides in a legacy identity store.
+:::
+
+## Automatic migration
+
+During automatic migration, Auth0 creates a new user in an identity store (database) managed by Auth0. Auth0 uses the identity in the Auth0 managed identity store when authenticating the user. For this to occur, Auth0 first requires the user be authenticated against the legacy identity store and only if this succeeds, will the new user be created in the Auth0 managed database. Auth0 creates the new user using the same id and password that was supplied during authentication. 
+
+::: Best Practice
+We recommend that you mark legacy store user identities once they have been migrated to Auth0 to prevent the unintentional re-creation of intentionally deleted users.
+:::
```

Comment:
We seem to be missing some of the original content defined here: https://docs.google.com/document/d/1a57Sj5z3WVfwBgNJ13ygbFtJfGfDx3SC67jryVnHYVc/edit#bookmark=id.xwhsw8vjo03z

Reply:
Added missing text.